GNUmakefile

```diff
@@ -31,6 +32,7 @@ bin: tools
 
 # dev creates binaries for testing locally - these are put into ./bin and $GOPATH
 dev:
+	vendorfmt
```

Comment:
This adds 1.4-2 sec to `make dev` for something your IDE usually does already. I don't think this would be a big win.